### DISCUSSION TITLE
Showing Error page on app.baseURL redirect

### DISCUSSION THREAD

**Discussion by doganalper** [nuxt/nuxt#16797] [Questions]
Hello everyone 👋🏼 
I have defined baseUrl using `app.baseURL` on nuxt.config file, it works normally. Now whenever I enter `localhost:3000/` it redirects me to `/baseURL` (as expected) but error page keeps still on page with custom 404 message.

**Comment by sevillaarvin** [comment]
> Now whenever I enter `localhost:3000/` it redirects me to `/baseURL` (as expected)

Hi, why do you think this is expected? I don't think it is expected to redirect. I think it is just the nuxt dev behavior, but for all intents and purposes, when you define a `baseURL` the website is only accessible through that path (unless you configure redirection in your web server, which is a separate topic altogether).
